src/horizontalaligner.cpp

```diff
@@ -461,17 +461,22 @@ void Alignment::ClearGraceAligners()
     m_graceAligners.clear();
 }
 
-std::pair<int, bool> Alignment::GetAccidAdjustedPosition(Doc *doc, int staffN)
+std::pair<int, bool> Alignment::GetAccidAdjustedPosition(Doc *doc, int staffN, int staffSize)
```

Comment:
Instead of the `std::pair` you could return just `int` with `VRV_UNSET` when nothing is found. But you can leave it like this.